src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -3,23 +3,24 @@
 import java.util.Random;
 
 /**
-* Class storing and managing the torpedoes of a ship
-*
-* (Deliberately contains bugs.)
-*/
+ * Class storing and managing the torpedoes of a ship
+ *
+ * (Deliberately contains bugs.)
+ */
 public class TorpedoStore {
+  private static Random generator = new Random();
```

Comment:
I would prefer a more descriptive variable name, sth like
```suggestion
  private static Random randomGenerator = new Random();
```